db/migrate/20240108145545_create_notification.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNotification < ActiveRecord::Migration[7.0]
+  def change
+    create_table :notifications do |t|
```

Comment:
we could add more restrictions on this db, at the moment it doesn't restrict things like start_date < end_date and all data fields are null-allowed

Reply:
also, not thrilled about saving html in the database, but the thinking (as I understand it) is that if we can't deploy we can still add a notification through the database if required, in which case this is necessary

Reply:
Hmm, yeah also not thrilled by html in the db but I can't really see a way round it if we want to be able to add notifications without a code release and we want to be able to put styling / headers / links in the pages.